### ISSUE TITLE
Android Autofill problems in Firefox

### ISSUE THREAD

**Issue by hpoul** [authpass/authpass#136]
Firefox correctly shows the `Autofill with AuthPass` prompt,  but once an entry is selected, it just disappears immediately.

![tempFileForShare_20200907-090633](https://user-images.githubusercontent.com/313066/92397097-7576e980-f126-11ea-8228-d398840eb1bf.jpg)

**Comment by hpoul** [comment]
This is weird.. in the android emulator everything works as expected 🤔️: (i've installed firefox through the play store as well..)

![2020-09-08 13-01-25 2020-09-08 13_06_46](https://user-images.githubusercontent.com/313066/92468932-4454f280-f1d4-11ea-9dd3-c048c9b1a618.gif)

**Comment by hpoul** [comment]
I did play around a bit more, but without too much luck. It's probably possible to work around this to at least sometimes make it work.. but i think it's a bug in Firefox. I have found which sounds related: https://bugzilla.mozilla.org/show_bug.cgi?id=1352011#c38 but has been open for years, does not look like to be too much of a priority.
especially since (most?) other browsers work i'm not sure if this is really something fixable on our side.

**Comment by luisantonpi** [comment]
Yep, and I can confirm myself that I've experienced this same bug myself more than a year later, on both ProtonMail's and Tutanota's login screens: https://emma.cloud.tabdigital.eu/s/Aa2g7jxBrAq9drF

**Comment by 14v** [comment]
Same behaviour like on @luisantonpi screen record on different login forms. After about 10 times clicking on fields autofill popup ends flickering and stays on screen able to be used.

**Comment by WesRos** [comment]
@hpoul 
I had the same. After make some changes in the app permissions, Authpass works as expected.

**Comment by pokatomnik** [comment]
The same problem. Latest ff, android 13. I see autofill suggestion, but no autofill after website choosing dialog.
